Broken release of the GraphQL Config

GraphQL config 3.0.1 stopped working from the source code:
graphql-config/config.d.ts(1,37): error TS2307: Cannot find module './types'.

https://circleci.com/gh/aerogear/graphback/4025?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ### vNEXT
 
+- Fix missing types [#542](https://github.com/kamilkisiela/graphql-config/issues/542)
+
 ### v3.0.1
 
 - use GraphQL Toolkit v0.10.6

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -2,11 +2,7 @@
   "compilerOptions": {
     "module": "esnext",
     "target": "es5",
-    "lib": [
-      "es2017",
-      "dom",
-      "esnext.asynciterable"
-    ],
+    "lib": ["es2017", "dom", "esnext.asynciterable"],
     "outDir": "dist",
     "declaration": true,
     "declarationMap": true,
@@ -16,11 +12,6 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true
   },
-  "exclude": [
-    "node_modules",
-    "dist"
-  ],
-  "files": [
-    "./src/index.ts"
-  ]
+  "exclude": ["node_modules", "dist"],
+  "include": ["src"]
 }

